grails-app/controllers/com/unifina/controller/api/AuthApiController.groovy

```diff
@@ -86,6 +92,7 @@ class AuthApiController {
 			def userProperties = [:] << cmd.properties << [
 				username: invite.email,
 				email: invite.email,
+				signupMethod: getSignupMethod()
 			]
 			user = userService.createUser(userProperties)
```

Comment:
There are other places where users are created too! This is only used by the legacy email/password signup flow. For example, if someone signs up with Ethereum, this controller is not involved. Those other places should be covered too.